pkg/controller/gc_controller.go

```diff
@@ -108,26 +108,36 @@ func (c *gcController) cleanBackups() {
 	// storage should happen first because otherwise there's a possibility the backup sync
 	// controller would re-create the API object after deletion.
 	for _, backup := range backups {
-		if backup.Status.Expiration.Time.Before(now) {
-			glog.Infof("Removing backup %s/%s", backup.Namespace, backup.Name)
-			if err := c.backupService.DeleteBackup(c.bucket, backup.Name); err != nil {
-				glog.Errorf("error deleting backup %s/%s: %v", backup.Namespace, backup.Name, err)
-			}
+		if !backup.Status.Expiration.Time.Before(now) {
```

Comment:
Do you think checking for "expiration after now" is clearer than "expiration not before now"?